components/CardWithHeader.tsx

```diff
@@ -12,10 +12,12 @@ export default function CardWithHeader({
   return (
     <div
       className={
-        'divide-y divide-gray-200 overflow-hidden rounded-lg bg-white shadow' + ' ' + className
+        'divide-y divide-gray-200 overflow-hidden rounded-xl bg-white shadow' + ' ' + className
       }
     >
-      <h2 className="px-4 py-5 sm:px-6 block text-xl">{header}</h2>
+      <h2 className="p-7 sm:px-6 font-light text-xl bg-gradient-to-br from-indigo-400 to-indigo-900 inline-block text-transparent bg-clip-text">
+        {header}
+      </h2>
```

Comment:
(arbitrary place to comment)

It looks great! I like the theme, background gradient, and the mobile is way better. There are a couple things we should address in phase 2 like some of the text contrast to make it more readable and other small things. After going through the styling pass are there any other changes we should make in phase 2? It may be good to document them in this thread and then create an issue to tackle in phase 2.